src/Component/Best.jsx

```diff
@@ -1,14 +1,65 @@
-import React, {Component} from "react";
-
+import React, { Component } from "react";
+import axios from "axios";
+import moment from "moment";
+import CONSTANTS from "../Services/Hnapi";
+const { baseUrl } = CONSTANTS;
 export default class Best extends Component {
-    render() {
-        return (
+  constructor() {
+    super();
+  }
+  state = {
+    getBestItem: [],
+  };
+
+  componentDidMount() {
+    const promises = [];
+    axios
+      .get(`${baseUrl}/beststories.json`)
+      .then((result) => {
+        this.results = result.data.slice(0, 30);
+        this.results.forEach((element) => {
+          promises.push(
+            axios
+              .get(`${baseUrl}/item/${element}.json`)
+              .then((data) => data.data)
+          );
+        });
+        Promise.all(promises)
+          .then((stories) => {
+            this.setState({
+              getBestItem: stories.filter((s) => s),
+            });
+          })
+          .catch((err) => {
+            console.log(err);
+          });
+      })
+      .catch((err) => {
+        this.err = err;
+      });
+  }
+
+  render() {
+    return (
+      <div>
+        {this.state.getBestItem.map((bestStories) => (
           <div>
-               <p>hello world</p>
+            <a href={bestStories.url}>
+              {" "}
+              <h2>{bestStories.title}</h2>
+            </a>
+            <p> {bestStories.score} points</p>
+            <p> by {bestStories.by}</p>
+            <p> {moment.unix(bestStories.time).fromNow()}</p>
+            <p>
+              comments{" "}
+              {bestStories.kids && bestStories.kids.length > 0
```

Comment:
better way to handle this case is(ie,  case with 0 in ternary operator): 
`
{bestStories.kids && bestStories.kids.length > 0 && bestStories.kids.length}
`